src/server/controllers/match.controller.js

```diff
@@ -211,13 +222,16 @@ export function countExpectedExpertiseMatching(req, res, next) {
       })
         .sort({ stamp_login: -1 }).exec();
     })
-    .then((mentorList) => {
+    .then((initialUserList) => {
       return new Promise((resolve, reject) => {
-        let careerFilteredList = [];
-
-        mentorList.forEach((user) => {
-          if (isFitCareerFilter(user.career, req.body.career)) {
-            careerFilteredList.push(user);
+        let careerFilteredList = {
+          full: [],
```

Comment:
`full` is an array that contains all of the information for the user object.
`_id` only contains `_id` string. This is used for checking duplication of users.